### ISSUE TITLE
[Bug]: Fan card doesn't work with Xiaomi Air Purifier 2S

### ISSUE THREAD

**Issue by TeoTN** [piitaya/lovelace-mushroom#1135]
### Requirements

- [X] I have updated Mushroom to the latest available version
- [X] I did a search to see if there is a similar issue or if a pull request is open.

### Current Behavior

Whenever I try to change percentage on my Xiaomi Air Purifier 2S, it doesn't work, and an alert is shown:
```
Failed to call service fan/set_percentage. Entity fan.air_purifier does not support this service.
```

### Expected Behavior

Setting fan operation percentage when in favourite mode should work

### Context

YAML state
```yaml

```

### Environment

```markdown
- Browser: Edge (latest)
- HA Version: 2023.5.0
- Mushroom: 2.7.1
```

**Comment by henriklund** [comment]
Not having access to a 'Xiaomi Air Purifier 2S' myself, I wonder how you set the fan speed prior to using Mushroom? It sounds to me as if the fan.set_percentage service is not implemented on your entity (and therefore not something I would expect Mushroom to be able to fix).
In case there is a way to change speed on your fan (using some other method than the 'set_percentage'), it might be worth considering to create a [template entity](https://www.home-assistant.io/integrations/fan.template/) with the necessary features.

**Comment by TeoTN** [comment]
Yeah, I realised later on that it's probably due to the broken service and opened an issue home-assistant/core#94437 asking for a fix to the service.

However, currently the device has a separate entity for the favorite level, something along the lines of `number.air_purifier_favorite_level`, so I adjust the speed using a slider for the number entity. Maybe adding optional support for a separate level entity would do the thing?

**Comment by piitaya** [comment]
You can use the [number card](https://github.com/piitaya/lovelace-mushroom/blob/main/docs/cards/number.md) for the number entity. It's not planned to add multiple entities support to Mushroom card.

**Comment by TeoTN** [comment]
Well, yes, I can skip using the fan card and do a Frankenstein setup but it'd be still nice if you reconsidered the assumption that fan has single entity to control it all 🙏
